docs/src/components/pricing/PricingTable.tsx

```diff
@@ -18,19 +18,20 @@ const planInfo = {
   community: {
     color: 'green',
     title: 'Community',
-    description: 'Get all the MUI component libraries, MIT-licensed, open source, and open core.',
+    description:
+      'Get all the MIT-licensed MUI component libraries. It includes open source and open core code.',
   },
   pro: {
     color: 'blue',
     title: 'Pro',
     description:
-      'Unlock essential features for building data-rich applications with complex use cases.',
+      'The last React component library you will need. Replace your third party dependencies with MUI.',
```

Comment:
To talk to the pain point.

Reply:
I don't think that it would work for the first plan because the community is about covering the most common use cases, it's not about the long tail. I don't think that it would work for the Premium plan because it's about embedded data analytics inside apps. I have found this interesting for the Pro plan is perfect for it because this plan competes with individual MIT libraries but gains all its value for customers by allowing developers to replace external libraries (e.g. react-table, react-date-picker) with a single vendor: @mui.

Reply:
I'm taking a step back then. The change seems controversial, going back to the version we have in https://v5-0-6.mui.com/pricing/

```suggestion
      'Best for professional developers building enterprise or data-rich applications.',
```